index.html

```diff
@@ -4,37 +4,172 @@
+          <li id="htm">html</li>
+          <li id="cs">css</li>
+          <li id="js">javascript</li>
+      </ul>
+        <a href="#"><button type="button" class="box">
+            <p>See Project</p>
+        </button></a>
+      </section>
+    </section>
+  </main>
+  <main class="cards-2">
+    <section id="fourth-work">
+      <div class="snap">
+        <img src="images/fourth.png" width="311px" height="220px" class="snap">
+      </div>
+      <section class="all">
+        <h5 class="title">Multi-Post<br>Stories</h5>
+        <ul class="back-end">
+        <li id="canopy">CANOPY</li>
+        <li><img src="images/Counter.svg" class="dev">Back End Dev</li>
+        <li><img src="images/Counter.svg" class="dev">2015</li>
+        </ul>
+        <p class="des">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="code">
+          <li id="htm">html</li>
+          <li id="cs">css</li>
+          <li id="js">javascript</li>
       </ul>
+        <a href="#"><button type="button" class="box">
+            <p>See Project</p>
+        </button></a>
+      </section>
     </section>
   </main>
+  <article class="about">
+    <h2>
+      About<br>Myself
+    </h2>
+    <p>
+      Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <h3 id="con">LETS CONNECT</h3>
+    <ul class="socials">
+      <li><a href="#"><img src="images/Tweeter.png"></a></li>
+      <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
+      <li><a href="#"><img src="images/m.png"></a></li>
+      <li><a href="#"><img src="images/Github.png"></a></li>
+      <li><a href="#"><img src="images/tech.png"></a></li>
+    </ul>
+    <a href="#"><button type="button" id="resume" class="box">Get my resume</button></a>
+  <div class="skills">
+    <div class="fs">
+      <h6 id="la">Languages</h6>
+    <i class="fa fa-angle-down"></i>
+    </div>
+    <div class="pics">
+      <img src="images/Frame 9.png" alt="Javascript">
+      <img src="images/htm.png" alt="HTML">
+      <img src="images/Frame 11.png" alt="css">
+    </div>
+    </ul>
+    <h6 id="fra">Frameworks</h6>
+    <hr>
+    <h6 id="sk">Skills</h6>
+    <i class="fa fa-angle-right"></i>
+    <hr>
+  </article>
+  <section class="contact-me">
+  <h2>Contact me</h2>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  <input id="name" type="text" name="name" placeholder="Yeremias">
+  <input id="email" type="email" name="email" placeholder="natajayanj14@gmail.com">
+  <input id="text-area" type="text" name="text-area" placeholder="Write your message here">
+  <button id="submit" type="submit" value="Get in Touch">Get in Touch</button>
+  </section>         
```

Comment:
As requested by the previous reviewer please remove this section, as it is a part of your upcoming milestone, and will need a seperate pull request for review. :pray: